front/app/components/UI/Error/index.tsx

```diff
@@ -246,74 +244,74 @@ const Error = (props: Props) => {
           {showIcon && <ErrorIcon name="error" data-testid="error-icon" />}
 
           <ErrorMessageText data-testid="error-message-text">
-            {typeof text === 'string' ? (
-              <p>{text}</p>
-            ) : (
-              <Box py="16px">{text}</Box>
-            )}
-            {dedupApiErrors &&
-              isArray(dedupApiErrors) &&
-              !isEmpty(dedupApiErrors) && (
-                <ErrorList>
-                  {dedupApiErrors.map((error, index) => {
-                    // If we have multiple possible errors for a certain input field,
-                    // we can 'group' them in the messages.js file using the fieldName as a prefix
-                    // Check the implementation of findErrorMessage for details
-                    const errorMessage = findErrorMessage(
-                      fieldName,
-                      error.error
-                    );
-
-                    if (errorMessage) {
-                      // Variables for inside messages.js
-                      const payload = error?.payload ?? null;
-                      const value = error?.value ?? null;
-                      const row = error?.row ?? null;
-                      const rows = error?.rows ?? null;
-
-                      let values = {
-                        row: <strong>{row}</strong>,
-                        rows: rows ? <strong>{rows.join(', ')}</strong> : null,
-                        // eslint-disable-next-line react/no-unescaped-entities
-                        value: <strong>'{value}'</strong>,
-                      };
-
-                      values = payload ? { ...payload, ...values } : values;
-
-                      if (value || row || rows) {
+            <Box py="16px">
```

Comment:
Made the error component paddings consistent between regular and API error messages.